modules/etcd-defrag.adoc

```diff
@@ -7,6 +7,14 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
+For large and dense clusters, etcd can suffer from poor performance if the keyspace grows too large and exceeds the space quota. Periodically maintain and defragment etcd to free up space in the data store. Monitor Prometheus for etcd metrics and defragment it when required; otherwise, etcd can raise a cluster-wide alarm that puts the cluster into a maintenance mode that accepts only key reads and deletes.
```

Comment:
etcd-defrag is automatic in the etcd-operator at a recent release.

Reply:
Correct, bot manual and automatic defragmentation processes are mentioned

Reply:
Worth adding link to the document - https://docs.openshift.com/container-platform/latest/scalability_and_performance/recommended-host-practices.html#etcd-defrag_recommended-host-practices as well?